### ISSUE TITLE
Menu missing at 200%

### ISSUE THREAD

**Issue by quarterback** [18F/cms-hitech-apd#1665]
Increasing the text to 200% caused the side menu to disappear and there was no way to move quickly through sections. Menu reappears below 200% (Was able to replicate issue, menu returns at 170% and below)

**Comment by hbillings** [comment]
Fixed by #1770
